ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -1127,6 +1168,24 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
+		60967E071D76B56D00863853 /* ADAL Preprocessing */ = {
```

Comment:
I notice in the build log that, "Process Info.plist" always happens before the Run Script, even after I put Run Script as top as I can in the "Build Phase" setting.

But I will try to dig deeper to see whether there is a way to do it :)